examples/LLM_Workflows/NER_Example/run.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+
+import lancedb_module
+import ner_extraction
+
+from hamilton import driver, lifecycle
+
+
+def build_driver(adapter_list):
+    """Builds the driver with the necessary modules and adapters."""
+    dr = (
+        driver.Builder()
+        .with_config({})
+        .with_modules(ner_extraction, lancedb_module)
```

Comment:
I'm confused as to why these are lumped together. They share one node at the end but its confusing... We only call that node in one spot? 

Oh, it seems that they have shared resources (one way). 

IMO it is cleaner/easier to share if you break it up + it highlights the value of modularity + reuse for the blog post.

Reply:
not sure I follow?

Reply:
Specifically there are two drivers. They do two different things. One has a dependency on the other. The graph is the same/there is one `build_driver` function.

Feels like we should break it into two?